sharding/networksharding/peerShardMapper.go

```diff
@@ -132,11 +134,7 @@ func (psm *PeerShardMapper) getPeerInfoWithNodesCoordinator(pid core.PeerID) (*c
 		}, false
 	}
 
-	psm.mutEpoch.RLock()
```

Comment:
Maybe remove the mutex, epoch and the associated handlers?

Reply:
I left there if later it would be used

Reply:
later means never :)))